lib/msal-browser/src/telemetry/BrowserPerformanceClient.ts

```diff
@@ -4,25 +4,43 @@
  */
 
 import {
+    Constants,
+    InProgressPerformanceEvent,
+    IPerformanceClient,
     Logger,
+    PerformanceClient,
     PerformanceEvent,
     PerformanceEvents,
-    IPerformanceClient,
-    PerformanceClient,
-    IPerformanceMeasurement,
-    InProgressPerformanceEvent,
-    SubMeasurement,
     PreQueueEvent,
-    Constants,
+    SubMeasurement,
 } from "@azure/msal-common";
-import { BrowserPerformanceMeasurement } from "./BrowserPerformanceMeasurement";
 import { Configuration } from "../config/Configuration";
 import { name, version } from "../packageMetadata";
+import {
+    BROWSER_PERF_ENABLED_KEY,
+    BrowserCacheLocation,
+} from "../utils/BrowserConstants";
 
 export class BrowserPerformanceClient
     extends PerformanceClient
     implements IPerformanceClient
 {
+    private static PERF_MEASUREMENT_MODULE = (() => {
```

Comment:
Suggest making this an unexported function instead of `static` for better minification

Reply:
Or alternatively, run this in the constructor and set a flag on the instance

Reply:
Made it a standalone function.